client/components/date-picker/index.jsx

```diff
@@ -88,13 +89,13 @@ class DatePicker extends PureComponent {
 		return eventsInDay;
 	}
 
-	locale() {
-		const { moment } = this.props;
+	getLocaleUtils() {
+		const { moment, localUtils } = this.props;
```

Comment:
Is this var deliberately not called `localeUtils`?

Reply:
Great spot. Fixed!